dokku.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eu
+
+cd /tmp && wget https://github.com/casey/just/releases/download/1.40.0/just-1.40.0-x86_64-unknown-linux-musl.tar.gz && tar -xzf just-1.40.0-x86_64-unknown-linux-musl.tar.gz && chmod 555 just
+sudo mv /tmp/just /usr/bin/
+
+cd /workspaces/opencodelists
+cp dotenv-sample .env
+touch db.sqlite3 # workaround because build-dbs-for-local-development expects a db.sqlite3
+yes Y | just build-dbs-for-local-development nuclear
+
+docker-compose up -d
+
+docker exec -t dokku sh -c "dokku apps:create opencodelists"
+docker exec -t dokku sh -c 'dokku config:set opencodelists BASE_URLS="http://localhost:7000,http://127.0.0.1:7000" DATABASE_DIR="/storage" DATABASE_URL="sqlite:////storage/db.sqlite3" DJANGO_SETTINGS_MODULE="opencodelists.settings" SECRET_KEY="thisisatestsecretkeyfortestingonly" TRUD_API_KEY="thisisatesttrudkeyfortestingonly"'
+
+sudo mkdir -p /var/lib/dokku/data/storage/opencodelists
```

Comment:
thought: I got very, very confused with permissions because I overlooked the `docker-compose.yaml` volumes configuration.

With the Docker Compose set up, this directory is on the host running Dokku in a container, not inside the Dokku container! (Inside the OpenCodelists container, I was seeing an empty directory owned by root.)